### DISCUSSION TITLE
Performance over 80K, I want to check if I got correct result. Is it normal?

### DISCUSSION THREAD

**Discussion by wtwinlab** [relab/hotstuff#131] [Q&A]
Hi, 

I run the following config:
run --consensus chainedhotstuff --max-concurrent 10000 --replicas 4 --clients 4 --batch-size 1000

Got result:
Throughput (commands/second),Latency (ms)
81483.81220586089,49.68059287697701

I want to check if it is normal. What's the reason that made the performance so high? Thanks.

**Comment by johningve** [comment]
Batch size of 1000 means that each block will contain 1000 client commands. Your result of 80000 commands / second means that the protocol completes 80 blocks per second. Seems reasonable to me.
